src/SDFileLogger.h

```diff
@@ -140,10 +163,44 @@ class SDFileLogger final : public LoggerBase
 		log_buffer_.reset();
 	}
 
+	void writeReadyBufferToSDFile()
```

Comment:
If this is the exact same function as the writeBufferToSDFile(), why not just pass in the buffer to be written as a parameter?

Reply:
Created template function for writeBufferToSDFile